turkeyvisited.js

```diff
@@ -26,11 +65,7 @@ d3.json("tr-cities.json").then(function (data) {
     .attr("d", path)
     .attr("fill", function (d, i) {
       if (localStorage.getItem("selectedCities")) {
-        if (
-          JSON.parse(localStorage.getItem("selectedCities")).includes(
-            d.properties.name
-          )
-        ) {
+        if (JSON.parse(localStorage.getItem("selectedCities")).includes(d.properties.name)) {
```

Comment:
- Ben kodlar üzerinde çalışırken böyle daha anlaşılır bulduğum için `if` bloğunun şartını tek satıra indirmiştim. İşlev olarak değişen birşey yok.